tests/output/staging/60027-UNI-21506.cpp

```diff
@@ -9,5 +9,5 @@ void Class::Foo(void (*callback)(const Class& entry))
 
 void foo()
 {
-    bool a = 1;  // if you comment this out, the bug stops reproducing
+    int a = 1;  // if you comment this out, the bug stops reproducing

```

Comment:
Why change the type to int?

Reply:
I think it's good. This is the output file. The input file was with `int`.

Reply:
👍